src/main/java/com/fasterxml/jackson/dataformat/xml/XmlFactory.java

```diff
@@ -325,6 +336,14 @@ public int getFormatGeneratorFeatures() {
         return _xmlGeneratorFeatures;
     }
 
+    public XmlTagProcessor getXmlTagProcessor() {
+        return _tagProcessor;
+    }
+
+    public void setXmlTagProcessor(XmlTagProcessor _tagProcessor) {
```

Comment:
Unless something in setup requires this, let's leave out this mutator: being a new feature should be possible to just use Builder approach. 3.0 (master) will have to remove it otherwise.